sonar-plugin-api/src/main/java/org/sonar/api/test/TestCase.java

```diff
@@ -23,8 +23,9 @@
 import javax.annotation.Nullable;
 
 /**
- * Represents a simple test execution result.
+ * @deprecated since 5.2 not used
  */
+@Deprecated
 public interface TestCase {
```

Comment:
![New](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/new.png)![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png)Do not forget to remove this deprecated code someday.[![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)